desc/coils.py

```diff
@@ -146,21 +147,21 @@ def current(self, new):
         assert jnp.isscalar(new) or new.size == 1
         self._current = float(new)
 
-    def compute_magnetic_field(self, coords, params=None, basis="rpz", grid=None):
+    def compute_magnetic_field(self, grid, params=None, basis="rpz", source_grid=None):
         """Compute magnetic field at a set of points.
 
         The coil current may be overridden by including `current`
         in the `params` dictionary.
 
         Parameters
         ----------
-        coords : array-like shape(n,3) or Grid
-            coordinates to evaluate field at [R,phi,Z] or [x,y,z]
-        params : dict, optional
-            parameters to pass to curve
+        grid : array-like shape(n,3) or Grid
```

Comment:
personally I prefer to not use `grid` here since that usually means a grid in computational coordinates but I could be convinced otherwise

Reply:
I second the hesitation, I prefer `coords` as it makes it clear it is real space coordinates that are being used versus rho,theta,zeta or `s` or some other computational coordinate, we are pretty consistent with this elsewhere in the code

Reply:
I like `eval_grid`, since that clarifies the difference from `source_grid` and I think that's consistent with our free-boundary notation. Also down the road we have discussed generalizing Grids to be in other coordinate systems. Is everyone good with that?

Reply:
Yes but on the free boundary branch `eval_grid` is always in computational coordinates, so I think its still a bit confusing here.

At the very least we should make it clear that it should be a regular `ndarray`, not a `Grid`, as currently `Grid`s are limited to computational coordinates.